Apps/Games/digital-extinction.toml

```diff
@@ -0,0 +1,4 @@
+name = "Digital Extinction"
+description = "3D real-time strategy game"
```

Comment:
I won't block on this, but I think you'll do better with a more detailed and evocative description :) This is your genre, but what's your hook?

Reply:
You are right. I've tried something better.